### PR TITLE
Synaptor PR part 1/3: General DAG changes

### PR DESCRIPTION
**Pull request by nicholasturner1**
This PR contains the set of commits that potentially affect the other DAG functionality within seuron. Part 2 will add the synaptor DAGs, and part 3 will add the slackbot changes.

Within part 1, there are three changes:
* Adding more arguments to the `worker_op` operator function.
* Adding a set of default parameters for synaptor to `dags/param_default.py`. These are currently not used, but may be with further updates.
* Adding a modified version of the `kombu_helper.py` file to `dags`. Using these functions as PythonOperators seemed like a cleaner way to offer a `drain_messages` operator than importing the chunkflow dags. I'm open to changing this if you want.

### CODE REVIEW

File: slackbot/slack_bot.py [hunk 1]
```diff
@@ -369,7 +369,6 @@ def dispatch_command(cmd, msg):
         update_inference_param(msg)
     elif cmd.startswith("cancelrun"):
         token = get_variable("run_token")
-        print(f"run_token: {token}")
```
Comment (ranlu) [inline slackbot/slack_bot.py]: Can you squash with the previous commit?

File: dags/worker_op.py [hunk 2]
```diff
@@ -17,4 +17,5 @@ def worker_op(**kwargs):
         execution_timeout=kwargs.get("execution_timeout", None),
         queue=kwargs["queue"],
         dag=kwargs["dag"],
+        qos=kwargs.get("qos", True),
```
Comment (ranlu) [inline dags/worker_op.py:20]: What does "qos" do?

File: dags/kombu_helper.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                    sleep(30)
+                    continue
+                else:
+                    break
+            #print(f'message: {message.payload}')
+            message.ack()
+            num_drained += 1
+
+        simple_queue.close()
+        print(f"# tasks drained: {num_drained}")
```
Comment (ranlu) [inline dags/kombu_helper.py]: It seems this is very similar to the one used in slack bot, maybe we can merge the two and place the code under the `common` folder. No need to do it in this pr.

File: dags/kombu_helper.py [hunk 4]
```diff
@@ -3,33 +3,39 @@
...
-                    print(f'still {status["messages"]} messages left in the queue, sleep for 30s')
+                    print(
+                        f'still {status["messages"]} messages left in the queue, sleep for 30s'
+                    )
                     sleep(30)
                     continue
                 else:
                     break
-            #print(f'message: {message.payload}')
+            # print(f'message: {message.payload}')
```
Comment (ranlu) [inline dags/kombu_helper.py]: The changes above are purely cosmetic, correct? I think it should be split from the code changes the behavior of the dags, or squash it with the previous one.

**Review by ranlu — CHANGES_REQUESTED** [review]

File: dags/worker_op.py (hunk 2 above)
Comment (nicholasturner1) [reply dags/worker_op.py:20]: This is a switch turning the container timeout after 5 minutes on/off. 
https://github.com/seung-lab/seuron/blob/main/plugins/custom/docker_custom.py#L188-L190

File: dags/kombu_helper.py (hunk 3 above)
Comment (nicholasturner1) [reply dags/kombu_helper.py]: Putting it in a common folder sounds good. I didn't know that was an option.

File: dags/kombu_helper.py (hunk 4 above)
Comment (nicholasturner1) [reply dags/kombu_helper.py:38]: The "conn" -> "connstr" change is actually required. Airflow reserves that argument name and will fail the task if this is used as a python callable.

Comment (nicholasturner1) [reply dags/kombu_helper.py]: The conn -> connstr change is actually required. Airflow reserves that argument name and will fail the task if this is used as a python callable. I can edit the commit to split these.

Comment (ranlu) [reply dags/kombu_helper.py]: Yes, please. Actually the original code is pretty confusing, we should rename `conn` to something meaningful like `queue_url` or `broker_url`

Comment (ranlu) [reply dags/kombu_helper.py]: And the commit message is quite confusing, can you split it into a pure cosmetic commit, a rename variable commit and a qos commit

Comment (nicholasturner1) [reply dags/kombu_helper.py]: I've removed the cosmetic changes, simplified the history, and renamed this argument to `broker_url`.

**Pull request merged**